modules/xxhash/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+    "homepage": "https://github.com/Cyan4973/xxHash",
+    "maintainers": [
+        {
+            "github": "Cyan4973",
```

Comment:
@mmorel-35 Again, this probably should be you if you are actually the one maintaining this module in BCR?